test/es-module/test-esm-specifiers-both-flags.mjs

```diff
@@ -0,0 +1,16 @@
+import { mustCall } from '../common/index.mjs';
+import { exec } from 'child_process';
+import assert from 'assert';
+
+const expectedError =
+  'node: bad option: cannot use --es-module-specifier-resolution ' +
+  'and --experimental-specifier-resolution at the same time';
+
+const flags = '--es-module-specifier-resolution=node ' +
+              '--experimental-specifier-resolution=node';
+
+exec(`${process.execPath} ${flags}`, {
+  timeout: 300
+}, mustCall((error) => {
+  assert(error.message.includes(expectedError));
```

Comment:
This test is currently failing on Windows. Removing the `node: bad option: ` part of the `expectedError` may help to ensure only the relevant part of the match is checked.